sdcm/sct_runner.py

```diff
@@ -797,7 +797,7 @@ def _create_instance(self,
                                        "block-project-ssh-keys": "true",
                                        "ssh-keys": f"{self.LOGIN_USER}:{self.key_pair.public_key.decode()}",
                                    },
-                                   network_tags=["keep-alive"],
+                                   network_tags=["keep-alive", "sct-network"],
```

Comment:
btw. @fruch is it intended to set keep-alive as network tag?

Reply:
Yes, it's still how the GCE resource cleaner code is looking it up.

Once we'll update that code we'll be able to remove this network tag